### PR TITLE
3 rd junes code

### PR DESCRIPTION
**Pull request by Subhransupanda2000**

### CODE REVIEW

File: src/main/java/com/example/jspiderjava/PrimeNumber.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package com.example.jspiderjava;
+
+public class PrimeNumber {
+    public static void main(String[] args) {
+        int n = 13;
+        boolean result = isPrimeNumber(n);
+        System.out.println("Is " + n + " a prime number? " + result);
+    }
+
+    public static boolean isPrimeNumber(int n) {
```
Comment (rkpatra201) [inline src/main/java/com/example/jspiderjava/PrimeNumber.java:10]: Private method here

File: src/main/java/com/example/jspiderjava/SumOfDigits.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
+package com.example.jspiderjava;
+
+public class SumOfDigits {
+    public static void main(String[] args) {
+        int n = 12345;
+        int result = sumOfDigits(n);
+        System.out.println(result);
+    }
+
+    public static int sumOfDigits(int n) {
```
Comment (rkpatra201) [inline src/main/java/com/example/jspiderjava/SumOfDigits.java:10]: Private method here

**Review by rkpatra201 — CHANGES_REQUESTED** [review]

File: src/main/java/com/example/jspiderjava/ElementOuccrence.java [hunk 3]
```diff
@@ -0,0 +1,21 @@
+package com.example.jspiderjava;
+
+public class ElementOuccrence {
```
Comment (rkpatra201) [inline src/main/java/com/example/jspiderjava/ElementOuccrence.java]: Fix spell error in class name

File: src/main/java/com/example/jspiderjava/ElementOuccrence.java [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    public static void main(String[] args) {
+        int n = 10;
+        int result = ouccuranceOfelements(n);
+        System.out.println(result);
+    }
+    private static int ouccuranceOfelements(int n) {
```
Comment (rkpatra201) [inline src/main/java/com/example/jspiderjava/ElementOuccrence.java]: Add junit test cases instead calling from main

Comment (rkpatra201) [inline src/main/java/com/example/jspiderjava/ElementOuccrence.java]: Explain by taking example in comments si that I understand what is the input and output. How that output came

**Review by rkpatra201 — CHANGES_REQUESTED** [review]

File: src/main/java/com/example/jspiderjava/ElementOuccrence.java (hunk 4 above)
Comment (Subhransupanda2000) [reply src/main/java/com/example/jspiderjava/ElementOuccrence.java]: Here i took input n as 10 .so we have n*n*n here we will get 1000. Here we are counting zeros thats why will get result as 3

File: src/test/java/org/example/TestDigitToCount.java [hunk 5]
```diff
@@ -0,0 +1,15 @@
+package org.example;
+import com.example.jspiderjava.DigitToCount;
+import org.junit.Test;
+import java.util.List;
+import static org.junit.Assert.assertEquals;
+public class TestDigitToCount {
+    @Test
+    public void testDigitTocount() {
```
Comment (rkpatra201) [inline src/test/java/org/example/TestDigitToCount.java:7]: Why expected value is 3

**Review by rkpatra201 — CHANGES_REQUESTED** [review]

**Review by rkpatra201 — APPROVED** [review]

**Pull request merged**